client/me/privacy/main.jsx

```diff
@@ -135,9 +160,82 @@ const Privacy = createReactClass( {
 						</FormButton>
 					</form>
 				</Card>
+				<SectionHeader label={ translate( 'Legal' ) } />
+				<Card>
+					<p>
+						{ /* translators: this is a privacy-related document we can provide to our users */
```

Comment:
```suggestion
						{
```

(not really necessary)

Reply:
I don't like the way it looks either, but I think prettier may have done it. Either way, it can go away when I remove my useless translator code comments :)

Reply:
oh... I see that you weren't formatting source code but removing a redundant comment. thanks!